tests/Microsoft.Identity.Test.Unit.net45/CacheTests/TokenCacheTests.cs

```diff
@@ -957,6 +932,99 @@ public void TestCacheDeserializeWithoutServiceBundle()
             tokenCache.DeserializeMsalV3(new byte[0]);
         }
 
+        [TestMethod]
+        public void TestGetAccounts_AcrossCliendIds()
+        {
+            var serviceBundle = TestCommon.CreateDefaultServiceBundle();
+            ITokenCacheInternal cache = new TokenCache(serviceBundle);
+           
+
+        }
+
+        [TestMethod]
+        public void TestIsFociMember()
+        {
+            // Arrange
+            using (var harness = new MockHttpAndServiceBundle())
+            {
+                harness.HttpManager.AddInstanceDiscoveryMockHandler();
+                ITokenCacheInternal cache = new TokenCache(harness.ServiceBundle);
+                AuthenticationRequestParameters requestParams = harness.CreateAuthenticationRequestParameters(
+                    MsalTestConstants.AuthorityTestTenant,
+                    MsalTestConstants.Scope,
+                    account: MsalTestConstants.User);
+
+                // Act
+                bool? result = cache.IsFociMemberAsync(requestParams, "1").Result;
+
+                // Assert
+                Assert.IsNull(result, "No app metadata, should return null which indicates <uknown>");
+
+                ValidateIsFociMember(cache, requestParams,
+                    metadataFamilyId: "1",
+                    expectedResult: true, // checks for familyId "1"
+                    errMessage: "Valid app metadata, should return true because family Id matches");
+
+
+                ValidateIsFociMember(cache, requestParams,
+                    metadataFamilyId: "2",
+                    expectedResult: false, // checks for familyId "1"
```

Comment:
>// checks for familyId "1" [](start = 43, length = 26)

are all of these texts true and relevant?

Reply:
Almost :). This one should say that it returns false.